.github/workflows/reusable-mqt-core-update.yml

```
@@ -63,31 +63,37 @@ jobs:
       - name: Get the commit SHA of the latest tag of MQT Core
         id: get-latest-tag-sha
         run: |
           echo "Querying the tags of MQT Core..."
-          tags=$(curl -s -H "Authorization: token ${{ github.token }}" https://api.github.com/repos/munich-quantum-toolkit/core/tags)
+          tags=$(curl -s -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/munich-quantum-toolkit/core/tags)
           echo "tags=$tags"
           latest_tag_sha=$(echo $tags | jq -r '.[0].commit.sha')
           echo "latest_tag_sha=$latest_tag_sha"
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
           echo "latest_tag_sha=$latest_tag_sha" >> $GITHUB_OUTPUT
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)